src/tools/compiletest/src/runtest.rs

```diff
@@ -1493,14 +1493,22 @@ impl<'test> TestCx<'test> {
                 unexpected.len(),
                 not_found.len()
             ));
-            println!("status: {}\ncommand: {}", proc_res.status, proc_res.cmdline);
+            println!("status: {}\ncommand: {}\n", proc_res.status, proc_res.cmdline);
             if !unexpected.is_empty() {
-                println!("unexpected errors (from JSON output): {:#?}\n", unexpected);
+                println!("{}", "--- unexpected errors (from JSON output) ---".green());
```

Comment:
Question: should this also respect `NO_COLOR`?

Reply:
it does, the methods are a no-op if the stuff isn't set

Reply:
Ah right, there's the colored control.